FuelLabs/fuel-indexer
# enhancement: Add graphQL info output when indexer runs

Closes #766 

Fairly self-explanatory. 

To test run :
```
cargo run --bin fuel-indexer -- run --manifest packages/fuel-indexer-tests/components/indices/fuel-indexer-test/fuel_indexer_test.yaml
```
You should see the output 
```
2023-04-19T15:07:46.654923Z  INFO fuel_indexer::commands::run: 70: GraphQL Playground: http://localhost:29987/api/playground/fuel_indexer_test/index1
```
